packages/client/src/raffles/types/rafflesQuery.types.ts

```diff
@@ -0,0 +1,13 @@
+export type RafflesQuery = {
+  status:
```

Comment:
I think you can replace this with `${RAFFLE_STATUS}` to get the values of the enum, instead of manually repeating them.

Either that or assume `status: RAFFLE_STATUS` and the consumer would use the `RAFFLE_STATUS` enum directly, which probably makes more sense than using an "arbitrary" string.

Reply:
Yes, it should be `status: RaffleStatus` here (I am using the name in Pascal case that was suggested in another comment)

Reply:
done. pls check again